README.md

```diff
@@ -6,5 +6,11 @@ Fork and clone this repository.
 
 ### Questions
 1. Explain why Rails exists and what it's used for.
+
+Rails is a library of ruby code that is used as a frameowrk for building websites.
```

Comment:
For building web _applications_. It's not just HTML/CSS that we're dealing with here - Rails is helping to set up up with all the things our application needs, including handling requests, manipulating data, and sending responses.